cmd/neofs-node/container.go

```diff
@@ -521,3 +531,42 @@ func (x *morphContainerReader) GetEACL(id *cid.ID) (*eaclSDK.Table, error) {
 func (x *morphContainerReader) List(id *owner.ID) ([]*cid.ID, error) {
 	return x.lister.List(id)
 }
+
+type morphContainerInvalidator struct {
+	containers interface {
+		InvalidateContainer(*cid.ID)
+	}
+
+	eacls interface {
+		InvalidateEACL(*cid.ID)
+	}
+
+	lists interface {
+		InvalidateContainerList(*owner.ID)
+		InvalidateContainerListByCID(*cid.ID)
+	}
+}
+
+func (x *morphContainerInvalidator) InvalidateContainer(id *cid.ID) {
+	if x.containers != nil {
```

Comment:
Is it possible at all?

Reply:
When cache turned off, all morphContainerInvalidator fields are nil.

Reply:
Thx, I got it, in current PR we call invalidator's methods even if cache is disabled.